### DISCUSSION TITLE
Where can the taskchampion docs be accessed

### DISCUSSION THREAD

**Discussion by RedEtherbloom** [GothenburgBitFactory/taskwarrior#3309] [Community support]
Is there currently a way to access the taskchampion documentation online?

The [link](https://gothenburgbitfactory.github.io/taskwarrior/taskchampion/) in the README.md atm 404s.
Is it being served under some different path?

I'll open it up locally otherwise.

**Comment by ryneeverett** [comment]
> The link in the README.md atm 404s.

This is a good point. @djmitche Are Github Pages still not enabled?

**Reply by djmitche** [reply]
I think I can do that now. I've opened #3311 to track doing os.
